spec/unit/pdk/cli/build_spec.rb

```diff
@@ -67,13 +73,27 @@
           expect(mock_metadata_obj).to receive(:interview_for_forge!)
           expect(mock_metadata_obj).to receive(:write!).with('metadata.json')
         end
+
+        it 'submits the command to analytics' do
+          allow(mock_metadata_obj).to receive(:interview_for_forge!)
+          allow(mock_metadata_obj).to receive(:write!)
+
+          expect(analytics).to receive(:screen_view).with('build', cli_options: 'target-dir=redacted', output_format: 'default', ruby_version: RUBY_VERSION)
```

Comment:
I haven't really worked with rspec expectations and keyword arguments before, is the default behavior that additional keywords arguments are accepted or if we add another keyword arg later will that cause this to fail?

Reply:
Also maybe adopt the multi-line syntax you use in `convert_spec.rb` for these as well.

Reply:
Adding additional arguments later will cause this to fail (we'd have to use something like `hash_including` if we wanted it to be permissive). This does increase the maintenance burden a bit but increases the confidence of not letting any private data slip in to analytics.